lib/marketingcloudsdk/client.rb

```diff
@@ -123,6 +127,26 @@ def initialize(params={}, debug=false)
 				end
 			end
 
+			if application_type.to_s.strip.empty?
+				self.application_type = 'server'
+			end
+
+			if ['web', 'public'].include? application_type
+				if authorization_code.to_s.strip.empty? or redirect_URI.to_s.strip.empty?
+					raise 'authorization_code or redirect_URI is null: For Public/Web Apps, the authorization_code and redirect_URI must be passed when instantiating Client'
+				end
+			end
+
+			if application_type == 'public'
+				if id.to_s.strip.empty?
+					raise 'id is null: id must be passed when instantiating Client'
```

Comment:
Should unify exceptions messages to refer in a single way to the client id. Here id is used, on line 145, Client Id is used.